tests/unit/test_coco_format.py

```
@@ -1010,20 +785,6 @@ def test_can_report_invalid_polygon_less_than_3_points(self):
             self.assertIsInstance(capture.exception.__cause__.__cause__, InvalidAnnotationError)
             self.assertIn("at least 3 (x, y) pairs", str(capture.exception.__cause__.__cause__))
 
-    @mark_requirement(Requirements.DATUM_ERROR_REPORTING)
-    def test_can_report_invalid_image_id(self):
```

Why this test was removed?

---

Upstream's version of coco just logs a warning in that case. Do we need to raise an exception?

---

It must be controlled by the importer policy object.

---

added parameter for the importer for it